SharkCage/CageConfigurator/CageConfiguratorForm.cs

```diff
@@ -408,7 +408,9 @@ private void saveButton_Click(object sender, EventArgs e)
 
                     writer.WriteStartObject();
                     writer.WritePropertyName("application_name");
-                    writer.WriteValue(FileVersionInfo.GetVersionInfo(applicationPath.Text)?.FileDescription);
+                    var app_name = FileVersionInfo.GetVersionInfo(applicationPath.Text)?.FileDescription;
+                    app_name = app_name ?? Path.GetFileName(applicationPath.Text);
```

Comment:
This fixes the problem for new config files, for the old ones,  it still crashes. 

Is it possible to extract some more data of that json exception and react more gracefully (e.g. displaying "name not available" or something along those lines) when encountering nul daal in a configuration file?